validator/validator-main.protoascii

```diff
@@ -5308,16 +5308,61 @@ tags: {
   html_format: ACTIONS
   enabled_by: "transformed"
   tag_name: "I-AMPHTML-SIZER"
+  spec_name: "I-AMPHTML-SIZER-RESPONSIVE"
   explicit_attrs_only: true
   attrs: {
     name: "style"
-    mandatory: true
     blacklisted_value_regex: "!\\s*important"
     css_declaration: {
       name: "display"
       value_casei: "block"
     }
     css_declaration: { name: "padding-top" }
+    dispatch_key: NAME_DISPATCH
+    mandatory: true
+  }
+}
+
+tags: {
+  html_format: AMP
+  html_format: ACTIONS
+  enabled_by: "transformed"
+  tag_name: "I-AMPHTML-SIZER"
+  spec_name: "I-AMPHTML-SIZER-INTRINSIC"
+  explicit_attrs_only: true
+  attrs: {
+    name: "class"
+    value: "i-amphtml-sizer"
+    mandatory: true
+    dispatch_key: NAME_DISPATCH
+  }
+}
+
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "IMG"
+  spec_name: "IMG-I-AMPHTML-INTRINSIC-SIZER"
+  mandatory_parent: "I-AMPHTML-SIZER"
```

Comment:
@honeybadgerdontcare @Gregable I tried `I-AMPHTML-SIZER-INRTINSIC`, but it didn't work. Is there another way?

Reply:
Sure, no problem.

Reply:
Let me know when you'd like me to take another look at this PR now that b/139869098 is done.

Reply:
I've updated the PR to check for the intrinsic sizer parent.